lib/active_model_serializers/adapter/attributes.rb

```diff
@@ -34,7 +34,7 @@ def serializable_hash_for_single_resource(options)
       def resource_relationships(options)
         relationships = {}
         serializer.associations(@include_tree).each do |association|
-          relationships[association.key] = relationship_value_for(association, options)
+          relationships[association.key] ||= relationship_value_for(association, options)
```

Comment:
do we want to hold off on merging this PR until this issue is resolve, or should we go ahead and merge? personally, I think optimizations to associations should be a seperate PR :-)